recipe/meta.yaml

```diff
@@ -25,21 +25,11 @@ requirements:
     - python
     - pip
     - setuptools
-    - numpy >=1.11
-    - scipy
-    - pyqt
-    - pyqtgraph
-    - lmfit
-    - fabio
-    - pyfai
-    - pycifrw
-    - future
-    - scikit-image
-    - qtpy
+    - numpy
 
   run:
     - python
-    - {{ pin_compatible('numpy') }}
```

Comment:
Are you sure the recipe does not use numpy's ABI?

Reply:
Looks like this is needed. See https://github.com/Dioptas/Dioptas/blob/develop/setup.py#L3

Reply:
It seems to work without it.

The main reason i am playing with this right now is different numpy versions.

I would like to use conda constructor to generate an installer from the conda-forge package.

The major problem is, that all other dependencies like fabio, pyfai, lmfit, can use the numpy from the anaconda distribution using intel-openmp. For some reason, Dioptas needs the normal openmp package from conda forge. Thus, it tries to install both and basically fails because there are two openmp libraries...

Reply:
What i actually find really annoying is the following:

having a blank python 3.6 environment, and do the following:

conda install -c conda-forge numpy

it will install me the conda-forge version of numpy.

If I do:

conda install -c conda-forge scipy

it will use the anaconda mkl version.

how do i prevent the mkl version and force conda to use the conda-forge version?

Reply:
> It seems to work without it.

Until a numpy is update/changed due to other requirements and you get a broken package. Numpy ABI must be respected unless you use a tight pin of building and running with the same numpy version.

There are many ways to avoid the issue you are mentioning above. From removing `defaults` from your builds, installations, constructors, etc to using the `nomkl` option. Note that, on Windows, `conda-forge` does not provide a `scipy` so the `defaults` version with `mkl` may be unavoidable there.

Still, what you are doing here **will** provide a broken package and bad experiences to the users of this package. The fact that it works under your scenario is not a reason to break the numpy ABI compatibility pinning.

Reply:
Ok, I just realized that the major problem is, that there is no scipy conda-forge version for windows, therefore conda always has to fallback on the base repository...

Reply:
Thanks for the comment. As said in the commit, this was a try to fix this. However, there is actually no way to fix it... 

There is no "nomkl" package on windows further there is no scipy on conda-forge for windows. What I do not understand for the 
{{ pin_compatible('numpy') }}
line

why is it not needed in the host but only for run?

Reply:
See https://conda-forge.org/docs/meta.html#building-against-numpy